source/atlas-cli-local-cloud.txt

```diff
@@ -0,0 +1,100 @@
+.. _atlas-cli-local-cloud:
+
+=========================================================
+Manage Local and Cloud Deployments from the {+atlas-cli+}
+=========================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use the {+atlas-cli+} to work with |service|, including using |fts| 
+and performing vector searches, throughout the entire software 
+development lifecycle, from your local environment to the cloud.
+
+.. include:: /includes/fact-atlas-deployments-preview.rst
+
+With the ``atlas deployments`` commands, you can perform the following
+actions with the {+atlas-cli+}:
+
+- Create local MongoDB deployments. Local deployments reside on your
+  computer and provide a non-production environment for development.
+- Create |service| cloud deployments. |service| hosts deployments in
+  the cloud for both non-production and production use.
+- Manage local {+clusters+} that you deployed with ``atlas
+  deployments``.
+- Manage all MongoDB |service| cloud deployments.
+
+.. note::
+
+   For local MongoDB deployments, you can manage only deployments that
+   you created with ``atlas deployments``. If you created a local
+   MongoDB {+cluster+} without using the {+atlas-cli+}, you can't
+   manage that {+cluster+} with the {+atlas-cli+}.
+
+Only ``atlas deployments`` {+atlas-cli+} commands work on local MongoDB
+deployments. All {+atlas-cli+} commands, including ``atlas
+deployments`` commands, work on MongoDB |service| cloud deployments.
+
+Supported Actions
+-----------------
+
+The actions you can perform with ``atlas deployments`` include but
+aren't limited to:
+
+- **Set up** a new local or |service| cloud {+cluster+}, including:
```

Comment:
@boooczek are we going to name them local Atlas deployments from now on instead of MongoDB local deployments? In places like setup we ask " How do you want to setup your local MongoDB database? "

Reply:
We're fine to use two forms: "local Atlas deployment" and "local Atlas database"